hr_holidays_natural_period/models/hr_leave_allocation.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2024 APSL-Nagarro - Antoni Marroig Campomar
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class HrLeaveAllocation(models.Model):
+    _inherit = "hr.leave.allocation"
+
+    type_request_unit = fields.Selection(
```

Comment:
Why adding this? I don't think it's needed.

Reply:
Because in V16 it was a `related` field, but now it is `computed` and can take its value from the `hr.leave.type's` `request_unit` field. However, options from the Selection are not inherited (as occurred in V16).
See https://github.com/odoo/odoo/blob/54edf00c6ad81f8143983820ecb43501199998fe/addons/hr_holidays/models/hr_leave_allocation.py#L347

Reply:
OK, please include this explanation summarized as a comment before the field declaration for future references.

Reply:
Done